core/assets/currencies.go

```diff
@@ -98,8 +99,13 @@ func (l *Link) Text(base int) string {
 	return (*big.Int)(l).Text(base)
 }
 
+var linkFmtThreshold = (*Link)(new(big.Int).Exp(big.NewInt(10), big.NewInt(12), nil))
+
 // MarshalText implements the encoding.TextMarshaler interface.
 func (l *Link) MarshalText() ([]byte, error) {
+	if l.Cmp(linkFmtThreshold) >= 0 {
+		return []byte(fmt.Sprintf("%s link", decimal.NewFromBigInt(l.ToInt(), -18))), nil
+	}
```

Comment:
I believe the smallest atomic unit of Link is called a Juel. Perhaps we want to use this explicitly to clarify?

Reply:
Yeah I meant to circle back here after hesitating to change the output and break compatibility. Any concern about re-using this type and changing the output? I'll double check the usages.